lib/logic/models/playlist.dart

```diff
@@ -1,14 +1,15 @@
 import 'package:audio_service/audio_service.dart';
 import 'package:sweyer/sweyer.dart';
+import 'package:sweyer_plugin/sweyer_plugin.dart';
 
-class Playlist extends PersistentQueue with DuplicatingSongOriginMixin {
+class Playlist extends PersistentQueue with DuplicatingSongOriginMixin, MediaStorePlaylist {
```

Comment:
> we shouldn't mix-in classes

What's the reason for not using mix-in classes?

Reply:
Ah, thanks for the explanation. You are right, I wanted to use interfaces here, but I just forgot that `implements` exists for that so I landed on the mixin. 😄 Will change to an interface.